contracts/test/helpers/ProposalTest.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I'm seeing a mix of MIT and BSD licenses

Reply:
For code where we enhanced existing OZ governor functionality, used MIT.
For places where we created code base on existing contracts that were BSD, used BSD.
This basically meant that the new file `IComp.sol` got changed from MIT to BSD.
Done: 633fea667b8da2e94e20f066493dab8815aedfe5